components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -321,7 +321,25 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
      */
     fun migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String) {
         rustCallWithLock { e ->
-            LibFxAFFI.INSTANCE.fxa_migrate_from_session_token(this.handle.get(), sessionToken, kSync, kXCS, e)
+            LibFxAFFI.INSTANCE.fxa_migrate_from_session_token(this.handle.get(), sessionToken, kSync, kXCS,
+                    false, e)
+        }
+        this.tryPersistState()
+    }
+
+    /**
+     * Copy a logged-in session of a Firefox Account, takes ownership of the provided session token.
```

Comment:
The docstring for `copy` says "takes ownership of the provided session token", I think maybe it got swapped around with the one for `migrate`?